api/hardwaremanagement/v1alpha1/firmwarecatalog_webhook.go

```
@@ -0,0 +1,111 @@
+	}
+
+	return nil, nil
+}
+
+// ValidateDelete implements webhook.CustomValidator
+func (v *firmwareCatalogValidator) ValidateDelete(_ context.Context, _ runtime.Object) (admission.Warnings, error) {
+	return nil, nil
+}
+
+// findRemovedEntries returns the names of entries present in old but absent from new.
+func findRemovedEntries(old, new []FirmwareImage) []string {
```

golangci-lint (2.12.2)

[error] param new has same name as predeclared identifier

(predeclared)